vendor/src/github.com/docker/distribution/registry/client/repository.go

```diff
@@ -345,9 +484,14 @@ func (ms *manifests) Delete(dgst digest.Digest) error {
 	if SuccessStatus(resp.StatusCode) {
 		return nil
 	}
-	return handleErrorResponse(resp)
+	return HandleErrorResponse(resp)
 }
 
+// todo(richardscothern): Restore interface and implementation with merge of #1050
```

Comment:
Sorry also didn't realize this was in vendor, apologies carry on :) also I like the factor out of code


On Jan 9, 2016, 12:08 -0800, Aaron Lehmann<notifications@github.com>, wrote:
>  
> Invendor/src/github.com/docker/distribution/registry/client/repository.go(https://github.com/docker/docker/pull/18785#discussion_r49267714):
>  
> >}>>+// todo(richardscothern): Restore interface and implementation with merge of #1050
> >+// todo(richardscothern): Restore interface and implementation with merge of#1050(https://github.com/docker/docker/issues/1050)This should be done or removed no?
> This is future work. The implementation was originally stubbed out, but I asked for it to be removed until an actual implementation exists.
>  
> —
> Reply to this email directly orview it on GitHub(https://github.com/docker/docker/pull/18785/files#r49267714).
>  
>  
>